DigitalLearningSolutions.Data.Tests/DataServices/GroupsDataServiceTests.cs

```diff
@@ -2,6 +2,7 @@
 {
     using System;
     using System.Linq;
+    using System.Transactions;

```

Comment:
I've added a test for the GroupDelegates deletion. I have no idea how to go about testing the other new method, not sure we have a sensible way to select anything to check it.

Reply:
Good point. I think we have a few other tests that create their own methods for selecting stuff from the database specifically for the tests, e.g CourseContentTestHelper so it might be worth seeing if something like that makes it more feasible.